### DISCUSSION TITLE
Wide layout

### DISCUSSION THREAD

**Discussion by jtagcat** [thinkst/canarytokens#247] [Feature Requests]
Quick sketchy sketch:

![image](https://user-images.githubusercontent.com/38327267/132960741-a77b19bc-336f-45a4-b6e2-fc454b80c0cf.png)

some inspiration from https://devdocs.io

***

Closes #107

It would make it more functional in terms it's less to scroll through, and more discoverable (you see what's available).

It's likely not really used on small mobiles (aka exluding tablets, etc). In a pinch, desktop layout would work; though using the current selector, or adding a three-hamburger menu button.

**Comment by jayjb** [comment]
Hi @jtagcat,

Thanks for the suggestion. I'll bring it up with the UI folks and see what they say

**Comment by jayjb** [comment]
Closing this issue because the UI team does not like the wide layout. Thank you for the suggestion though. It is an interesting thought.
